.github/kokoro/steps/vtr-test.sh

```diff
@@ -68,8 +68,12 @@ find vtr_flow/benchmarks/titan_blif/ -type f -not -name 'README.*' -delete
 # Removing ISPD, Titan and Symbiflow tarballs
 find . -type f -regex ".*\.tar\.\(gz\|xz\)" -delete
 
+#Gzip output net and pack files from vtr_reg_nightly tests to lower working directory disk space
```

Comment:
Just net files, need to change comment

Reply:
Just a thought, maybe compress everything under `vtr_flow/tasks/regression_tests/vtr_reg_nightly/` and time how long it takes, e.g.:
```
date
find vtr_flow/tasks/regression_tests/vtr_reg_nightly/ -name *.net -print0 | xargs -0 -P $(nproc) gzip -v
date
```

?  If it's fast, I think it might be a good idea.

Reply:
Sure, I will try that